### PR TITLE
Live Push 2024-03-07: fix accessibility issues

### PR DESCRIPTION
**Pull request by EverStarck**
closes:  https://github.com/justia/justatic/issues/4805, https://github.com/justia/justatic/issues/4799

---

### Features

- Add `aria-hidden` to cta buttons (bcf68ff88d4d87177881808830074142dd51bde5)

### Bug Fixes

- Accessibility issue with button (f75acb8feef3d22fb3049bf8104775f898efedf8)
- BreakpointZone its undefined (5d93c436e288107e374dfc6b6eb5585884454456)

### Code Refactoring

- Change slides cta button helper name (bcfd8052d13254b449bdb5ac6ac881da84e43233)

### Internal Workflow

- Update dist files (3ae0d1f1fb0ee46f605d07e7bee6a535890c8faf)

 ---

I used the worst variable name possible, if you have a suggestion, I can change them.

### CODE REVIEW

File: src/tiny-slider.js [hunk 1]
```diff
@@ -1073,7 +1080,7 @@ export var tns = function(options) {
     // == controlsInit ==
     if (hasControls) {
       if (!controlsContainer && (!prevButton || !nextButton)) {
-        outerWrapper.insertAdjacentHTML(getInsertPosition(options.controlsPosition), '<div class="tns-controls" aria-label="Carousel Navigation" tabindex="0"><button type="button" data-controls="prev" tabindex="-1" aria-controls="' + slideId +'">' + controlsText[0] + '</button><button type="button" data-controls="next" tabindex="-1" aria-controls="' + slideId +'">' + controlsText[1] + '</button></div>');
+        outerWrapper.insertAdjacentHTML(getInsertPosition(options.controlsPosition), '<div class="tns-controls" aria-label="Carousel Navigation" tabindex="0"><button type="button" data-controls="prev" aria-label="Previously" tabindex="0" aria-controls="' + slideId +'">' + controlsText[0] + '</button><button type="button" data-controls="next" tabindex="0" aria-controls="' + slideId +'">' + controlsText[1] + '</button></div>');
```
Comment (priscilamtzfraire) [inline src/tiny-slider.js:1083]: This should be `Previous`, no? 🤔

Comment (EverStarck) [reply src/tiny-slider.js:1083]: lil-yachty-drake

File: src/helpers/getSlidesCTABtn.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export function getSlidesCTABtn (slideItems) {
+    return slideItems.map((item) => item.querySelector('.ui-button.-cta') || false)
```
[the remark this answers is not in the document]
Comment (EverStarck) [reply src/helpers/getSlidesCTABtn.js]: fixed: [0c32c1e](https://github.com/justia/tiny-slider/pull/11/commits/0c32c1eb089bc347c71ada0d4e2524a250bfbdb2)

File: src/tiny-slider.js (hunk 1 above)
Comment (EverStarck) [reply src/tiny-slider.js]: fixed [df436f1](https://github.com/justia/tiny-slider/pull/11/commits/df436f16f9e6034411e18fdcf2317e7c1535252c)

**Review by priscilamtzfraire — APPROVED** [review]
thanks! 🚀

**Pull request merged**